internal/analytics/client.go

```diff
@@ -32,6 +33,7 @@ func (c *Client) SendEvent(
 	eventName string,
 	properties map[string]interface{},
 ) {
+	properties["id"] = c.ID
```

Comment:
In SendEvent we'll intercept  properties and add our id. This means that both CLI Command Run and CLI Command Completed would share the same id.